### PR TITLE
updated the locator file names

### PR DESCRIPTION
**Pull request by sgovindankutty**
# Critical Changes

# Changes

# Issues Closed

# Community Ideas Delivered

# Features Intended for Future Release

# Features for Elevate Customers

# New Metadata

# Deleted Metadata

### CODE REVIEW

File: robot/Cumulus/resources/NPSP.py [hunk 1]
```diff
@@ -33,13 +33,13 @@
...
+from locators_53 import npsp_lex_locators as locators_53
+from locators_54 import npsp_lex_locators as locators_54
 
 locators_by_api_version = {
-    52.0: locators_52,  # summer '21
-    51.0: locators_51,  # spring '21
-    50.0: locators_50   # winter '21
+    54.0: locators_54,  # summer '22
+    53.0: locators_53,  # spring '22
+    52.0: locators_52   # winter '21
```
Comment (npsp-reedestockton) [inline robot/Cumulus/resources/NPSP.py:42]: API 52 was winter '22.  Also, now that API 55 is out, shouldn't we be using locators_55 as the highest API version??

Comment (sgovindankutty) [reply robot/Cumulus/resources/NPSP.py:42]: done!

File: robot/Cumulus/resources/locators_53.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+"""Locators for spring '20"""
```
[the remark this answers is not in the document]
Comment (sgovindankutty) [reply robot/Cumulus/resources/locators_53.py:1]: done!

**Review by npsp-reedestockton — APPROVED** [review]
Approved!  Thanks @sgovindankutty!!!

**Comment by lparrott** [comment]
Superseded by #7041

**Pull request closed without merging**